### DISCUSSION TITLE
Grafana dashboard for actions-metrics-server

### DISCUSSION THREAD

**Discussion by martindelisletsfx** [actions/actions-runner-controller#2685] [Questions]
Hi!

If anyone could please share their dashboard json here to monitor from the actions-metrics-server it would be much appreciated. I am having a hard time deriving "status over time" types of metrics from the histograms. I see there is most likely all the info we need but to be honest, a few of us had a look at the metrics and we couldn't untie them properly.

Any help would be welcome!

**Comment by Samsanga** [comment]
@martindelisletsfx , Did you manage to get `actions-metrics-server` working on GitHub Enterprise or GitHub.com ?

**Reply by martindelisletsfx** [reply]
github.com, we use actions-runner-controller to run our own self-hosted-runners but our github repos are all on github.com.
